packages/dashboards/tests/acceptance/mutli-datasource-dashboards-test.js

```diff
@@ -24,8 +24,14 @@ module('Acceptance | Multi datasource Dashboard', function (hooks) {
     await click('.dashboard-filters__toggle');
 
     assert.deepEqual(
-      findAll('.filter-builder__subject').map((el) => el.textContent.trim()),
-      ['Age (id)', 'Container (id)'],
+      findAll('.filter-builder__subject .name').map((el) => el.textContent.trim()),
```

Comment:
Can we use `assert.dom` here

Reply:
do we want all of the similar instances changed to .dom?